vyxal/LazyList.py

```diff
@@ -165,9 +165,7 @@ def __next__(self):
         return item
 
     def __setitem__(self, position, value):
-        if position >= len(self.generated):
-            self.__getitem__(position)
-        self.generated[position] = value
+        raise NotImplementedError("LazyList does not support assignment")
 
     def __delitem__(self, index):
```

Comment:
This method can also be removed.

Reply:
My idea was that having an error would help make it clear if there were any errors trying to assign to a lazy lists.